modules/apps/josh-lapurd-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,775 @@
+				color: $theme-base-light
+			}
+		}
+
+		&.nav-bar-adjustment {
+				height: 80px;
+
+				#themeBanner {
+					padding: 30px 2% 0 2%;
+				}
+
+				#themeNavBar {
+					padding-top: 75px;
+				}
+		}
+	}
+
+	#themeNavBar {
+		background: $theme-bg-light-2;
+		bottom: 0;
+		box-sizing: border-box;
+		display: block;
+		height: 100%;
+		overflow-y: auto;
+		padding: 20px 0;
+		position: fixed;
+		right: -250px;
+		top: 0;
+
+		@include transition(all 0.5s ease-in-out);
+
+		width: 250px;
+		z-index: 200;
+
+		ul {
+			margin: 0;
+			padding: 0;
+		}
+
+		li {
+			border-bottom: 1px solid $default-color;
+			margin-left: 0;
+			margin-right: 0;
+			padding: 10px 0;
+			width: 100%;
+
+			.child-menu-toggle {
+				color: inherit;
+				cursor: pointer;
+				font-size: inherit;
+				height: 25px;
+				line-height: 1.3;
+				width: 25px;
+			}
+
+			li {
+				border-left: 1px solid $default-color;
+			}
+
+			&:hover {
+				background: $theme-bg-light-1;
+			}
+
+			ul.child-menu {
+				background: $theme-bg-light-2;
+				display: none;
+				margin: 0;
+				padding-left: 5%;
+				padding-right: 0;
+				position: relative;
+				width: 100%;
+
+				& ~ ul {
+					display: none;
+				}
+			}
+
+			& > ul {
+				position: relative;
+			}
+		}
+
+		a {
+			color: $default-color;
+			padding: 0 10px;
+		}
+
+		.selected {
+			background: $theme-base-light;
+		}
+
+		.theme-nav-bar-toggle-close {
+			font-size: 3em;
+			line-height: 0;
+		}
+
+		ul.child-menu-open {
+			background: inherit;
+			display: block !important;
+			margin: 0;
+			padding: 0;
+			position: relative !important;
+		}
+
+		button.child-menu-toggle-open {
+			transform: rotate(90deg);
+		}
+
+		&.menu-open {
+			right: 0;
+		}
+	}
+
+	#navigation > ul > li.selected .child-menu {
+		background-color: $theme-bg-light-2;
+	}
+}
+
+@include media-query($screen-md, null) {
+	#banner.nav-bar-adjustment {
+			top: 35px;
+	}
+
+	#navigation ul {
+		li.selected a:before {
+			width: 100%;
+		}
+
+		li:hover {
+			& > {
+				.child-menu {
+					background-color: $theme-bg-dropdown;
+					display: block;
+				}
+
+				a:before {
+					width: 100%;
+				}
+			}
+		}
+	}
+
+	#navigation > ul {
+		& > li {
+			margin: 0 10px;
+
+			& > a:before {
+				background-color: $theme-base;
+				bottom: 0;
+				content: '';
+				height: 4px;
+				left: 50%;
+				margin: 0 auto;
+				position: absolute;
+				top: auto;
+				transform: translate(-50%, -48px);
+				transition: width 0.3s ease 0s;
+				width: 0;
+			}
+		}
+
+		& > li:hover > ul {
+			display: block;
+			width: auto;
+		}
+
+		& ul > li:hover > ul {
+			display: block;
+			margin-left: 100%;
+			top: 0;
+			width: auto;
+			z-index: 200;
+		}
+
+		& > li .child-menu li {
+			white-space: nowrap;
+		}
+
+		& > li > ul .child-menu {
+			margin-left: 50%;
+		}
+	}
+
+	#navigation li button.child-menu-toggle {
+		display: none;
+	}
+
+	.grid-text {
+		h3 {
+			font-size: 24px;
+		}
+
+		p {
+			font-size: 15px;
+			color: $default-color
```

Comment:
Is this needed?  The ```p``` tags are already defined to use the ```$default-color``` at https://github.com/jonmak08/liferay-portal/pull/1719/files#diff-e5e6c4da8a59f74d3788916afebd2dcdR157.